host/hackrf-tools/src/hackrf_biast.c

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2012-2022 Great Scott Gadgets <info@greatscottgadgets.com>
+ * Copyright 2012 Jared Boone <jared@sharebrained.com>
+ * Copyright 2013 Benjamin Vernoux <titanmkd@gmail.com>
```

Comment:
None of these people wrote this code! You should put your own details here (and similarly on other files you've added).

Reply:
Forgetting to put your name on things is how you know you've been working on proprietary code way too long ;)